api/v1alpha1/rsyncnodedatamovement_types.go

```diff
@@ -51,6 +51,14 @@ type RsyncNodeDataMovementSpec struct {
 	// DryRun specifies that this data movement request should show what would have been transferred without actually
 	// performing any copy operation.
 	DryRun bool `json:"dryRun,omitempty"`
+
+	// Cancel is a flag to indicate that this data movement should be
+	// terminated. This value should not be set on creation.
+	Cancel bool `json:"cancel,omitempty"`
```

Comment:
This should have a default=False - take a look around nnf-sos for kubebuilder annotations (or google)